java/src/main/java/ai/rapids/cudf/Rmm.java

```diff
@@ -185,14 +185,30 @@ public static void resetScopedMaximumBytesAllocated() {
    * @throws RmmException if an active handler is already set
    */
   public static void setEventHandler(RmmEventHandler handler) throws RmmException {
+    setEventHandler(handler, false);
+  }
+
+  /**
+   * Sets the event handler to be called on RMM events (e.g.: allocation failure) and
+   * optionally enable debug mode (callbacks on every allocate and deallocate)
+   *
+   * NOTE: Only use this while debugging, code will run slower!
```

Comment:
This note implies the entire method should not be used or code will run much slower which is incorrect.
```suggestion
   * NOTE: Only enable debug mode when necessary, as code will run much slower!
```